repospanner.service

```diff
@@ -2,6 +2,9 @@
 Description=repoSpanner
 After=network.target
 
+[Install]
```

Comment:
Could you please split this out to a separate PR, and remove one of the two duplicate commits and the merge commit?